src/Application/FacetConfig.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseFacetedSearch\Application;
+
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\Entity\PropertyId;
+
+class FacetConfig {
+
+	public function __construct(
+		public readonly ItemId $itemId,
+		public readonly PropertyId $propertyId,
+		public readonly string $type
```

Comment:
This should be something more specific than a string. I don't think we need a full widget registry, and instead an enum will be fine.

Reply:
Agreed, we can use an Enum. No need to support adding widget types via extensions to WBFS